arcgis-ios-sdk-samples/Layers/Apply unique values with alternate symbols/ApplyUniqueValuesAlternateSymbolsViewController.swift

```diff
@@ -0,0 +1,118 @@
+// Copyright © 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ApplyUniqueValuesAlternateSymbolsViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Set the map and its initial viewpoint.
+            mapView.map = AGSMap(basemapStyle: .arcGISTopographic)
+            mapView.setViewpoint(AGSViewpoint(center: AGSPoint(x: -13632095.660131, y: 4545009.846004, spatialReference: .webMercator()), scale: 7500))
+        }
+    }
+    
+    @IBOutlet var currentScaleLabel: UILabel!
+    @IBOutlet var resetViewpointBarButtonItem: UIBarButtonItem!
+    
+    /// Response to the bar button item being tapped.
+    @IBAction func resetViewpointTapped(_ button: UIBarButtonItem) {
+        // Create the initial viewpoint.
+        let viewpoint = AGSViewpoint(center: AGSPoint(x: -13631205.660131, y: 4546829.846004, spatialReference: .webMercator()), scale: 7500)
+        // Set the viewpoint with animation.
+        mapView.setViewpoint(viewpoint, duration: 5, curve: AGSAnimationCurve.easeInOutSine) { (finishedWithoutInterruption) in
+            if finishedWithoutInterruption {
+                self.mapView.setViewpoint(viewpoint, duration: 5, curve: .easeInOutSine)
+            }
+        }
+    }
+    
+    /// The feature service URL.
+    static let featureServiceURL = URL(string: String("https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0"))!
+    /// The feature layer set in San Francisco, CA.
+    let featureLayer = AGSFeatureLayer(featureTable: AGSServiceFeatureTable(url: featureServiceURL))
+    
+    /// The formatter used to generate strings from scale values.
+    private let scaleFormatter: NumberFormatter = {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.maximumFractionDigits = 0
+        return numberFormatter
+    }()
+    
+    /// Creates the unique values renderer for the feature layer.
+    func createUniqueValuesRenderer() {
+        // Create the default symbol.
+        let symbol = AGSSimpleMarkerSymbol(style: .triangle, color: .red, size: 30)
+        // Convert the symbol to a multi layer symbol.
+        let multiLayerSymbol = symbol.toMultilayerSymbol()
+        multiLayerSymbol.referenceProperties = AGSSymbolReferenceProperties(minScale: 5000, maxScale: 0)
```

Comment:
Either do underscore or scientific notation, i.e., 5e3. Same for other occurrences.